aviary/wren/data.py

```diff
@@ -270,14 +269,14 @@ def parse_aflow_wyckoff_str(
     elements = []
     wyckoff_set = []
 
-    subst = r"1\g<1>"
-    for el, wyk in zip(elems, wyckoff_letters):
+    for el, wyk_letter in zip(elems, wyckoff_letters):
 
         # Put 1's in front of all Wyckoff letters not preceded by numbers
-        wyk = re.sub(r"((?<![0-9])[A-z])", subst, wyk)
+        if not wyk_letter[0].isdigit():
```

Comment:
Not sure about this I think wyk are all the wyks for an element i.e. multiple sites may need 1's in front of them hence regex

Reply:
should make a test example that has something like `abc_a_b:X-Y-Z` at end of wyk string o test correctly

Reply:
Yes, was about to say that.

Reply:
Resolved in a9644ed.